internal/backend/auth/authusers/users.go

```diff
@@ -0,0 +1,91 @@
+package authusers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"go.uber.org/zap"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+	"go.autokitteh.dev/autokitteh/internal/backend/db"
+	"go.autokitteh.dev/autokitteh/sdk/sdkerrors"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+type Config struct {
+	TestUsersEmails string `koanf:"test_users_emails"`
```

Comment:
for testing. problem is that currently we don't have an externally accessible API to add users. will do so when we have some minimal RBAC. i think we should leave this until then. lmk if it doesn't make sense to you.

Reply:
in testing you have full access to db, 
I prefer having a testing migration that runs in tests mode instead of adding test code to production 
what do you think ?